core/src/main/scala/cromwell/core/WorkflowMetadataKeys.scala

```diff
@@ -37,4 +37,6 @@ object WorkflowMetadataKeys {
   val Labels = "labels"
   val MetadataArchiveStatus = "metadataArchiveStatus"
   val Message = "message"
+
+  val CommandLine = "commandLine"
```

Comment:
Is this used anywhere?

Reply:
No, I added that in a previous iteration -- its been removed.